models/classes/class.QtiTestService.php

```diff
@@ -615,17 +626,25 @@ protected function importTest(
         // -- Check if the file referenced by the test QTI resource exists.
         if (is_readable($expectedTestFile) === false) {
             $report->add(
-                common_report_Report::createFailure(
+                Report::createError(
                     __('No file found at location "%s".', $qtiTestResource->getFile())
                 )
             );
         } else {
+            //Convert to QTI 2.2
+            if ($this->getPsrContainer()->has(TestConverter::class)) {
```

Comment:
I don't see any checks to validate we are dealing with QTI3 import package, we don't need it? It will not break QTI2 packages?